### PR TITLE
2.37.0-rc.1

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.37.0-rc.0
+2.37.0-rc.1
```
Comment (Nexucis) [inline VERSION:1]: is it normal that you have a "." between `rc` and `1` ? On other rc we don't have that

**Comment by roidelapluie** [comment]
2.34.0-rc.0
2.33.0-rc.0 
2.32.0-rc.0
2.31.0-rc.0
2.30.0-rc.0


2.35 and 2.36 are the different ones

File: VERSION (hunk 1 above)
Comment (roidelapluie) [reply VERSION:1]: 2.34.0-rc.0
2.33.0-rc.0
2.32.0-rc.0
2.31.0-rc.0
2.30.0-rc.0

2.35 and 2.36 are the different ones

**Review by Nexucis — APPROVED** [review]
🤞

**Comment by juliusv** [comment]
:+1: :crossed_fingers:

**Comment by roidelapluie** [comment]
I am running into interesting bugs about oauth2, I will work a couple of hours and see if we require an additioal bugfix in this RC

**Review by hdost — COMMENTED** [review]
👍🏼

**Pull request merged**